src/Compilers/VisualBasic/Test/Semantic/BasicCompilerSemanticTest.vbproj

```diff
@@ -220,5 +91,21 @@
   <ItemGroup>
     <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
   </ItemGroup>
+  <ItemGroup>
+    <Content Include="Semantics\BinaryOperatorsTestSource1.vb" />
+    <Content Include="Semantics\BinaryOperatorsTestSource2.vb" />
+    <Content Include="Semantics\BinaryOperatorsTestSource3.vb" />
+    <Content Include="Semantics\BinaryOperatorsTestSource4.vb" />
+    <Content Include="Semantics\BinaryOperatorsTestSource5.vb" />
+    <Content Include="Semantics\OverloadResolutionTestSource.vb" />
+    <Content Include="Semantics\PrintResultTestSource.vb" />
+    <Compile Remove="Semantics\BinaryOperatorsTestSource1.vb" />
+    <Compile Remove="Semantics\BinaryOperatorsTestSource2.vb" />
+    <Compile Remove="Semantics\BinaryOperatorsTestSource3.vb" />
+    <Compile Remove="Semantics\BinaryOperatorsTestSource4.vb" />
+    <Compile Remove="Semantics\BinaryOperatorsTestSource5.vb" />
+    <Compile Remove="Semantics\OverloadResolutionTestSource.vb" />
+    <Compile Remove="Semantics\PrintResultTestSource.vb" />
```

Comment:
Sure would be nice to not have to specify `<Compile Remove` for `<Content Include` items.